### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by bast**
Work in progress.

### CODE REVIEW

**Comment by bast** [comment]
This is coupled to https://github.com/coderefinery/coderefinery.org/pull/283.

File: accounts.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+A couple of accounts are needed during the course. The motivation
+for this is to provide a real life hands-on experience using tools with massive
+traction and network effect.
+
+We recommend that all participants who do not yet have user accounts on the following
+platforms create new accounts before the workshop starts:
+
+- [GitHub](https://github.com/)
+- [Read the Docs](https://readthedocs.org/)
```
Comment (blankdots) [inline accounts.md]: people can sign in into read the docs and travis with github, so they only need a github account

File: git.md [hunk 2]
```diff
@@ -81,7 +81,27 @@ and [configure Git](#configuring-git).
...
+and [configure Git](#configuring-git).
+
+
+### Installation using Conda
+
+This assumes you have already installed Anaconda. Then you can install Git in one line:
+
+```shell
+$ conda install git
+```
```
Comment (wikfeldt) [inline git.md]: with this approach shouldn't one be using anaconda prompt rather than git-bash?

Comment (bast) [reply git.md]: Yes exactly. Should I clarify this better?

File: accounts.md (hunk 1 above)
Comment (bast) [reply accounts.md]: Indeed, this will simplify things. I will send a patch for this.

File: git.md (hunk 2 above)
Comment (bast) [reply git.md]: Or asking differently: I don't see the contradiction/problem. But I can imagine this is not clear enough.

Comment (wikfeldt) [reply git.md]: i'll look at the whole material for better overview and see if this might be misunderstood, brb

Comment (wikfeldt) [reply git.md]: ok it's clear that GitBash and conda are two alternative methods to get Git, but maybe clarify by writing "Then you can install Git in one line in the Anaconda Prompt:"

**Comment by bast** [comment]
> great work, i approve. i can clarify that point on Anaconda Prompt in a follow-up PR

Thanks! I will send a patch to add your suggested sentence (you can also suggest changes directly in the review which I can commit via button) while also adjusting the text on Read the Docs account, later today.

File: accounts.md (hunk 1 above)
Comment (bast) [reply accounts.md]: Done.

File: git.md (hunk 2 above)
Comment (bast) [reply git.md]: Hopefully clearer now.

**Comment by bast** [comment]
I am merging since the chance increases by the day that participants already start to install software.

**Pull request merged**